calcom/cal.com
# Fix attendees on a booking with seats and payments, navigate to the payment page.

## What does this PR do?



This PR fixes the bug where not all attendees on a booking with seats and payments were not taken to the payment page.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Create an event type with seats and a price
- Book as the first attendee. You should be taken to the payment page as a part of the booking
- Book the same time slot as another attendee. You should also be taken to the payment page

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
